src/DacpacTool/BuildOptions.cs

```diff
@@ -16,5 +16,8 @@ public class BuildOptions : BaseOptions
         public FileInfo PreDeploy { get; set; }
         public FileInfo PostDeploy { get; set; }
         public FileInfo RefactorLog { get; set; }
+
+        public bool WarnAsError { get; set; }
+        public string Suppress { get; set; }
```

Comment:
I don't think the name `Suppress` here is clear enough, I'd prefer `SuppressWarnings`